src/main/java/uk/gov/ida/verifyserviceprovider/configuration/EuropeanIdentityConfiguration.java

```diff
@@ -54,6 +64,11 @@ public String getHubConnectorEntityId() {
                 .orElse(environment.getEidasHubConnectorEntityId());
     }
 
+    public String[] getAcceptableHubConnectorEntityIds() {
+        return Optional.ofNullable(acceptableHubConnectorEntityIds)
```

Comment:
Why are there two placed to obtain that value....? This doesn't seem right and is confusing.

Reply:
This might be a good question to ask to the original author.

Reply:
🤦‍♂

Reply:
Can we make it better?

Reply:
I've added `acceptableHubConnectorEntityIds` wherever `hubConnectorEntityId` is defined.

`hubConnectorEntityId` is defined in `EuropeanIdentityConfiguration` (apparently constructed from json), and `HubEnvironment` (an enum with a lot of hard coded params). There's very likely a good reason why it's the way it is. It may take some time to figure out. Happy to look into it.